cmd/pulumi-test-language/interface_test.go

```diff
@@ -91,7 +91,15 @@ func TestUniqueProviderVersions(t *testing.T) {
 			vstr := version.String()
 
 			if v, ok := versions[vstr]; ok {
-				assert.Equal(t, pkg, v, "provider version %s is used by both %s and %s", vstr, pkg, v)
+				assert.Equal(
+					t,
+					pkg,
+					v,
+					"provider version %s is used by both %s and %s, next provider version is 16.0.0",
```

Comment:
Can we not hardcode 16.0 Just loop over everything first to find the max version and then +1

Reply:
Also if you split this into it's own PR we can just merge it in quick